### ISSUE TITLE
Can't set as wallpaper a file whose name contains double dash

### ISSUE THREAD

**Issue by KuttKatrea** [l3ib/nitrogen#77]
If you try to set the wallpaper to a file having double dash in its name, it will fail:

    $ nitrogen --ecca2167ef7c5f217b5b239ca64e6bf3.png
    Error parsing command line: Unexpected argument ecca2167ef7c5f217b5b239ca64e6bf3.png

    $ nitrogen ecca2167ef7c5f2--17b5b239ca64e6bf3.png
    Error parsing command line: Unexpected argument 17b5b239ca64e6bf3.png

It is an edge case when you have a filename starting with double dash, but having them in the middle of the name would not cause any problem

Using `1.5.2-2build1` in Ubuntu Xenial

**Comment by daf** [comment]
Fair enough - this is a pretty uncommon situation though.  We do use our own argument parser from years back so it doesn't understand common argument passing practices like using a ` -- ` to indicate "end of options", and maybe we'll have to support this.  Will add to a future milestone.
